notebooks/basics/single-systems.ipynb

```diff
@@ -88,26 +85,26 @@
     "### 1.1 Classical states and probability vectors <a id='classical-states'></a>\n",
     "\n",
     "Suppose that we have a [system](gloss:system) that stores information.\n",
-    "More specifically, we will assume that this system can be in one of a finite number of *classical states* at each instant.\n",
-    "Here, the term classical state should be understood in intuitive terms, as a configuration that can be recognized and described unambiguously.\n",
+    "More specifically, let's assume that this system can be in one of a finite number of *classical states* at each instant.\n",
+    "Here, the term \"classical state\" should be understood in intuitive terms, as a configuration that can be recognized and described unambiguously.\n",
     "\n",
-    "The archetypal example, which we will come back to repeatedly, is that of a bit, which is a system whose classical states are 0 and 1.\n",
+    "The archetypal example, which we will come back to repeatedly, is that of a *bit*, which is a system whose classical states are 0 and 1. Any system with two states provides a realization of a classical bit: the labels are unimportant.\n",
     "Other examples include a standard six-sided die, whose classical states are 1, 2, 3, 4, 5, and 6; a nucleobase in a strand of DNA, whose classical states are *A*, *C*, *G*, and *T*; and a switch on an electric fan, whose classical states are (commonly) *high*, *medium*, *low*, and *off*.\n",
     "In mathematical terms, the specification of the classical states of a system are, in fact, the starting point: we define a bit to be a system that has classical states 0 and 1, and likewise for systems having different classical state sets.\n",
     "\n",
     "For the sake of this discussion, let us give the name $\\mathsf{X}$ to the system being considered, and let us use the symbol $\\class{_sigma}{\\Sigma}$ to refer to the set of classical states of $\\mathsf{X}$.\n",
-    "As well as the assumption that $\\Sigma$ is finite, as mentioned earlier, we naturally assume that $\\Sigma$ is nonempty: it is nonsensical for a physical system to have no states at all that it can be in.\n",
-    "Although it does make sense to consider physical systems having infinitely many classical states, we will ignore this possibility for now.\n",
+    "As well as the assumption that $\\Sigma$ is finite, as mentioned earlier, we naturally assume that $\\Sigma$ is nonempty: it is nonsensical for a physical system to have no states at all.\n",
+    "Although it does make sense to consider physical systems having infinitely many classical states, we will ignore this possibility for now. There are plenty of interesting ideas to explore even within only a finite number of states.\n",
```

Comment:
The sentence "There are plenty..." may give the wrong impression. We're not really limiting ourselves or making due with just a finite number of states — I think it's correct to say that _most_ of the interest in quantum information is focused on the finite state case.

Reply:
Would you consider saying something along those lines? Maybe, instead of the last sentence, "This is hardly a restriction, since the vast majority of quantum information theory deals with finite state sets." My thinking is it could be helpful to justify this choice and assuage any doubt.